Source/WPF/MyMoney/Utilities/StringHelpers.cs

```diff
@@ -141,6 +142,21 @@ public static string CamelCase(this string input)
             }
             return sb.ToString();
         }
+
+        public static CultureInfo TryToGetCulureDefaultBackToUS(string cultureName)

```

Comment:
Can you add a comment, I cannot figure out what this method is trying to do given the name, this name makes no sense to me.  Or perhaps change it to this if this is what you intended:
```c#
public static CultureInfo GetCultureInfo(string cultureName, fallback="en-US"){
   ... same code but using the fallback
}
```
Note it is no longer a "TryGet" because it always returns something, TryGet methods usually return a boolean and an optional out parameter.

Reply:
Good susgestion

Reply:
fixed